doc/Overview/Mvux/Feeds.md

```diff
@@ -0,0 +1,259 @@
+---
+uid: Overview.Mvux.Feeds
+---
+
+# What are Feeds?
+
+Feeds are there to manage asynchronous data requests from a service and provide their result to the View in an efficient manner.
+
+It provides out of the box support for data coming from task-based methods as well Async-Enumerables ones.  
+
+They accompany the requests with additional metadata that tell whether the request is still in progress, ended in an error, and if it was successful, whether the data that was returned contains any entries or was empty.
+
+## Feeds are stateless
+
+Feeds are used as a gateway to request data from services, but it does not hold the data. The data is rather sent over to the requesting party awaiting the feed itself (see [below](#directly-await-feeds)).
+
+As such, feeds do not provide support for reacting upon changes the user makes to the data, and the data is preserved in its original state.  
+You can look at a Feed as a read-only representation of the requested data.
+
+> [!TIP]
+> In contrast to Feeds, [States](xref:Overview.Mvux.States) are stateful and keep track of the up-to-date state as applied by changes from the View by the user.
+
+## How to use Feeds?
+
+### Creation of Feeds
+
+For the examples below let's use a counter service that returns the current count number, starting from 1. It will be run 3 consecutive times delayed by a second each.
+For the data type we'll create a record type called `CounterValue`:
+
+```c#
+public record CounterValue(int Value);
+```
+
+Feeds can be created directly from either `ValueTask` returning methods, or from `IAsyncEnumerable` methods,
+both with a `CancellationToken` parameter.
+
+#### Using Tasks
+
+Asynchronous data can be obtained in several ways.
+
+The most common is via a `ValueTask` that returns the data value(s) when ready:
+
+```c#
+_currentCount = 0;
+public async ValueTask<CounterValue> CountOne(CancellationToken ct)
+{
+    await Task.Delay(TimeSpan.FromSeconds(1));
+
+    // note that a service does not normally hold data
+    // this example is for demonstration purposes
+    return new CounterValue(++_currentCount);
+}
+```
+
+> [!NOTE]
+> `ValueTask` is interchangeable with `Task`, but `ValueTask` was chosen to be in unity with the `IAsyncEnumerable` interface.
```

Comment:
Perhaps link to a doc that discusses differences between Task and ValueTask for those not familiar with ValueTask

Reply:
The only source I found was an article by Stephen Taub (tho official but not part of the docs).
Will try to find something in the docs, otherwise will link to article.

Reply:
Added.